model/metabolites.tsv

```diff
@@ -8455,3 +8455,9 @@ mets	metsNoComp	metBiGGID	metKEGGID	metHMDBID	metChEBIID	metPubChemID	metLipidMa
 "MAM01435m"	"MAM01435"	""	"C02528"	"HMDB0000518"	"CHEBI:16755"	"10133"	"LMST04010032"	""	"HC00958"	"HC00958"	"MNXM1183"	""	"MAM03318m"
 "MAM01729e"	"MAM01729"	""	""	"HMDB0002250"	""	"168381"	""	""	""	"M01729"	"MNXM51468;MNXM588354"	""	"MAM03540e"
 "MAM00853c"	"MAM00853"	""	""	""	""	""	""	"CE4820"	""	"CE4820"	"MNXM164259"	""	";MAM00902c"
+"MAM20077m"	"MAM20077"	""	"C2267"	""	"CHEBI:29144"	"5460642"	""	""	""	""	"MNXM37367"	""	""
```

Comment:
Is C2267 a KEGG ID?

Reply:
ah that's supposed to be [C22667](https://www.kegg.jp/entry/C22667)